pkg/rsnotify/listeners/postgrespgx/pgxlistener.go

```diff
@@ -226,10 +226,10 @@ func (l *PgxListener) notify(n *pgconn.Notification, errs chan error, items chan
 	}
 	t, err := l.matcher.Type(dataType)
 	if err != nil {
+		errs <- fmt.Errorf("no matcher type found for %d", dataType)
```

Comment:
Here, and also in the pqlistener, if we switch to error wrapping, we can simply pass the error here:
```
errs <- err
```